src/main/java/com/springweb/dv_spring_web_mongo/configuration/Config.java

```diff
@@ -1,19 +1,22 @@
-package com.springweb.dv_spring_web_mongo;
+package com.springweb.dv_spring_web_mongo.configuration;
 
 import com.mongodb.ConnectionString;
 import com.mongodb.client.MongoClient;
 import com.mongodb.MongoClientSettings;
 import com.mongodb.client.MongoClients;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
 import org.springframework.data.mongodb.core.MongoTemplate;
 
 @Configuration
-public class config {
+@ComponentScan(basePackages = "com.springweb.dv_spring_web_mongo")
```

Comment:
I think your application can work without this annotation because @SpringBootApplication provides it